### PR TITLE
Add isActive flag to vtgr to support multi-cell topology

### PR DESCRIPTION
**Pull request by sonne5**
Signed-off-by: Yang Wu <y.wu4515@gmail.com>

## Description

When we deploy VTGR in more than one region, we usually deploy one mysql group per region to avoid cross-region paxos round. 

Because of that, we need to define an "active" region for VTGR: only VTGR in the active region will do the `tmc.ChangeType` call. VTGR in the inactive region will just maintain the mysql group without modifying vttablet records in topo server

See the diagram in #8779 for more illustration

## Related Issue(s)

Resolves https://github.com/vitessio/vitess/issues/8779

## Checklist
- [ ] Should this PR be backported?
- [x] Tests were added or are not required
- [x] Documentation was added or is not required

### CODE REVIEW

**Comment by sonne5** [comment]
Reviewers: please review with `Hide whitespace changes` enabled

File: go/vt/vtgr/controller/repair.go [hunk 1]
```diff
@@ -308,7 +308,7 @@ func (shard *GRShard) getGTIDSetFromAll(skipPrimary bool) (*groupGTIDRecorder, *
 	primary := shard.findShardPrimaryTablet()
 	var mysqlPrimaryHost string
 	var mysqlPrimaryPort int
-	// skipPrimary is true when we manual failover or if there is a unreachalbe primary tablet
+	// skipMaster is true when we manual failover or if there is a unreachalbe primary tablet
```
Comment (deepthi) [inline go/vt/vtgr/controller/repair.go:311]: have to change this back.

**Review by deepthi — COMMENTED** [review]
LGTM. We can merge after the review is addressed.

File: go/vt/vtgr/controller/repair.go (hunk 1 above)
Comment (sonne5) [reply go/vt/vtgr/controller/repair.go]: Yep, done

**Review by deepthi — APPROVED** [review]

**Pull request merged**